lib/common/templates/flutter/class.dart.template

```diff
@@ -0,0 +1,41 @@
+<%
+//
+// Copyright 2019 Alaska Air Group or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License").
+// You may not use this file except in compliance with the License.
+// A copy of the License is located at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+%>
+//
+// <%= this.destination %>
```

Comment:
This was an iOS pattern for files, we might not need it for Flutter projects.

Reply:
I actually used the same pattern for flutter on my apps, and it met my needs. If someone come with a more suitable pattern for flutter, we can change it.